### ISSUE TITLE
Location is estimated despite not having enough recordings above cc_threshold

### ISSUE THREAD

**Issue by rhine3** [kitzeslab/opensoundscape#930]
In my localization I'm requiring at least 4 receivers for localization. I'm finding that when I set a cross-correlation threshold and only 2 receivers meet it (i.e., the reference recording with a cross-correlation of 1 and one other recording), OpenSoundscape still returns a localization. 

Some of the relevant code is here but I can't immediately find the bug:

https://github.com/kitzeslab/opensoundscape/blob/cf77a561f8b7f372b930c928b07349af958490a8/opensoundscape/localization.py#L235-L253

I'm not sure where the flaw in the logic is, but when I inspect these events, their `receivers_used_for_localization` attribute only lists the 2 recordings that had an above-threshold cross correlation

**Comment by louisfh** [comment]
Good catch! The problem was the way `localize` is written (the wrapper for gillette and soundfinder localization algorithms), it will return a 'position estimate' even if there are fewer tdoas than it is possible to estimate a location from.
